autopush-common/src/db/dynamodb/mod.rs

```diff
@@ -524,6 +525,13 @@ impl DbClient for DdbClientImpl {
             )
             .await?;
 
+        let metric = self.metrics.incr_with_tags("notification.message.stored");
+        if let Some(topic) = topic {
+            metric.with_tag("topic", topic.as_str()).send();
+        } else {
+            metric.send();
+        }
+
```

Comment:
You want to render true/false like the autoconnect code does,
```suggestion
        self.metrics.incr_with_tags("notification.message.stored")
            .with_tag("topic", message.topic.is_some().to_string()).send()
 ```